internal/model/marshaler.go

```diff
@@ -22,6 +22,8 @@ import (
 
 // TracesMarshaler marshals pdata.Traces into bytes.
 type TracesMarshaler interface {
+	// Marshal the given pdata.Traces as bytes.
```

Comment:
Nit: Marshal into

Marshal the given pdata.Traces into bytes.

Reply:
Done.